### DISCUSSION TITLE
Can't create new react app.

### DISCUSSION THREAD

**Discussion by Ryan-Tung** [sst-inc/help#61] [Q&A - React] [answered]
![image](https://user-images.githubusercontent.com/88029861/151093718-01660a62-389d-47d3-a7f9-b0aa64c2c2f5.png)
When I tried the commands given to fix issues, 
![image](https://user-images.githubusercontent.com/88029861/151093782-9bbbacbd-5fd7-4290-ba87-697bfa6a886b.png)
It did not work.

**Comment by ghost** [comment, marked as answer]
You are already in the ios folder, so just skip the `cd ios` command.
